### DISCUSSION TITLE
blog/highlight-lines-without-breaking-the-code-in-xaringan/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#124] [Comments]
# blog/highlight-lines-without-breaking-the-code-in-xaringan/

Data Scientist. #rstats. Educator. Musician. Always learning something new.

https://www.garrickadenbuie.com/blog/highlight-lines-without-breaking-the-code-in-xaringan/

**Comment by jorgesinval** [comment]
I get a strange behavior of the highlights when using Sideways Panelsets together with your source code language next to code boxes and insert the name of the chunk.
Let me exemplify:
1 - without naming the chunk:
![The highlights (using #<<) work as expected](https://i.imgur.com/6MoQ2yF.png)

2 - naming the chunk:
![The highlight do not work as expected](https://i.imgur.com/9aNgjE7.png)

I believe that is something related with the use of special symbols as `  or *

Additionality, if I use the `*`  symbol:

![I get the code in italics](https://i.imgur.com/NjEq96z.png)
